CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ in the naming of release branches.
 
 ### Added
 
+- Add ToExpr (from tree-diff) instances for all types inside NewEpochState
+  - Add module Cardano.Ledger.TreeDiff in cardano-binary. Includes all needed operations, classes, and orphan instances	
+- Add deposits (key deposits) to DState{dsDeposits} and (pool deposits) PState{psDeposits}
+  - Added property tests that test the new invariants that must hold
+    - utxosDepoits == sum (dsDeposited) + sum (psDeposited)
```

Comment:
A little typo here, missing an `s` in `utxosDeposits`

Reply:
good catch.